zenml/core/steps/tokenizer/hf_tokenizer.py

```diff
@@ -0,0 +1,247 @@
+#  Copyright (c) maiot GmbH 2021. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+
+from typing import Dict, Text, List, Any
+
+import tensorflow as tf
+
+from zenml.core.steps.tokenizer.base_tokenizer import BaseTokenizer
+from zenml.core.steps.split.utils import get_categorical_value
+from zenml.core.steps.tokenizer.utils import tokenizer_map
+from zenml.utils import path_utils
+
+
+class TokenizerStep(BaseTokenizer):
```

Comment:
Should we call this a `HuggingFaceTokenizerStep`?

Reply:
Sure

Reply:
Solved in 140d160